client/components/Card/Card.jsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react'
 import pt from 'prop-types'
 
+import { useSelector, useDispatch } from 'react-redux'
+
+import { updateCartProducts, createCart } from '../../redux/cart/actions'
+// import { createCart } from '../../api/cart'
 import './Card.scss'
 
 const Card = ({ item }) => {
-  const { name, image, price, description } = item
+  const { name, image, price, description, _id } = item
+
+  const dispatch = useDispatch()
+  const loading = useSelector((state) => state.cart.loading)
+  const cartId = useSelector((state) => state.cart.id)
+
+  const handleClick = (e, productId) => {
+    e.preventDefault()
+
+    if (loading) return
```

Comment:
@AStashevskaya Lets use `{` and `}` braces, even if statement contains single instructions, it much easier to add debug info, or other instructions in this case:

```js
if (loading) {
  return
}
```